commons/src/main/java/org/infinispan/commons/util/concurrent/ConcurrentHashSet.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.util.concurrent;
+package org.infinispan.commons.util.concurrent;
```

Comment:
@slaskawi We could keep the copy in commons, but it should still be in a separate commit/JIRA.